python/pfnet/cpfnet.pyx

```diff
@@ -897,6 +934,11 @@ cdef class Bus:
                 g = cvargen.VARGEN_get_next(g)
             return vargens
 
+    property var_gens:
+        """ List of :class:`variable generators <pfnet.VarGenerator>` connected to this bus (list). """
```

Comment:
See point 4) I think we should just say "Same as :data:`var_generators <pfnet.Bus.var_generators>`" (or something like that). I think this is better since it tells you that this is just an alias